Run-time dependency wlroots found: NO (tried pkgconfig and cmake)
Looking for a fallback subproject for the dependency wlroots

meson.build:45:11: ERROR: Subproject exists but has no meson.build file.

--- meson.build.orig	2024-08-24 11:04:54 UTC
+++ meson.build
@@ -42,11 +42,11 @@ elif get_option('use_system_wlroots').enabled()
 
 elif get_option('use_system_wlroots').enabled()
 	use_system_wlroots = true
-	wlroots = dependency('wlroots', version: ['>=0.17.0', '<0.18.0'], required: true)
+	wlroots = dependency('wlroots-0.17', version: ['>=0.17.0', '<0.18.0'], required: true)
 
 elif get_option('use_system_wlroots').auto()
 	message( 'SEARCHING FOR WLROOTS' )
-	wlroots = dependency('wlroots', version: ['>=0.17.0', '<0.18.0'], required: false)
+	wlroots = dependency('wlroots-0.17', version: ['>=0.17.0', '<0.18.0'], required: false)
 	use_system_wlroots = true
 	if not wlroots.found()
 		use_system_wlroots = false
--- src/meson.build.orig	2024-08-24 11:04:54 UTC
+++ src/meson.build
@@ -147,7 +147,7 @@ public_api_requirements = [
     wayland_server,
     pixman,
     # These might be subprojects so we need to pass them as strings
-    'wlroots',
+    'wlroots-0.17',
     'wf-config',
 ]
 
